ui/src/utils/sort.js

```diff
@@ -20,8 +20,8 @@ const sort = (referenceElement, compareElement) => {
     return diff < 0 ? -1 : 1;
   }
 
-  if (Date.parse(referenceElement) || Date.parse(compareElement)) {
-    return compareDesc(referenceElement, compareElement);
+  if (isDate(referenceElement) && isDate(compareElement)) {
```

Comment:
Navigating to http://localhost:5080/provisioners/aws-provisioner-v1/worker-types/gecko-1-b-linux throws `Invalid time value`. I wonder if this is because we are changing `||` to `&&` here.

Reply:
Found out what the issue is. When a task is running, its task resolved date is `null`. With the newer version of `date-fns`, it throws an error now when date is invalid.
In this case, do you want to display `n/a`,similar to quarantined date?
Currently on master, it displays invalid date:
![image](https://user-images.githubusercontent.com/1489148/62676410-c3944900-b978-11e9-93ec-864c89b21017.png)

Reply:
Good catch! `n/a` sounds good to me :)